test/runtests.sh

```diff
@@ -1,28 +1,8 @@
-#-------------------------------------------------------------------------------------------------------
 # Copyright (C) Microsoft. All rights reserved.
 # Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 #-------------------------------------------------------------------------------------------------------
 #
-# Eventual test running harness on *nix for ChakraCore
-# Today, it's simply there to make sure hello.js doesn't regress
-#
+# REMOVE THIS AFTER MERGING runtests.py in!
```

Comment:
The idea is that, since netci.groovy is much slower to update and the way we run the tests could change, I'm ultimately trying to reduce resistance to changes by keeping this file as a wrapper for whatever mechanism we actually use to run tests. I think that's a fair enough technical reason (based on a technical limitation of the `netci.groovy` task generation mechanism.

I'm not aware of any decision regarding `netci.groovy` task changes or this file in particular. I'll start an email thread with @curtisman to sort out the details.